src/zimlet/com_zimbra_birthdayreminder/com_zimbra_birthdayreminder_vi.properties

```diff
@@ -0,0 +1,79 @@
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Zimlets
+# Copyright (C) 2010, 2013, 2014, 2016 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+#
+
+label = Tr\u00ecnh nh\u1eafc sinh nh\u1eadt.
+description = Qu\u00e9t S\u1ed5 \u0111\u1ecba ch\u1ec9 \u0111\u1ec3 xem ng\u00e0y sinh nh\u1eadt v\u00e0 t\u1ea1o l\u1eddi nh\u1eafc l\u1ecbch.
+
+BirthdayReminder_CalendarName = Nh\u1eafc nh\u1edf sinh nh\u1eadt
+
+#
+# Zimlet messages
+#
+BirthdayReminder_label = Tr\u00ecnh nh\u1eafc sinh nh\u1eadt.
+BirthdayReminder_panelTooltip = Qu\u00e9t S\u1ed5 \u0111\u1ecba ch\u1ec9 \u0111\u1ec3 xem ng\u00e0y sinh nh\u1eadt v\u00e0 t\u1ea1o l\u1eddi nh\u1eafc l\u1ecbch.
+
+#
+# Error messages
+#
+BirthdayReminder_couldNotCreateCalendar = Kh\u00f4ng th\u1ec3 t\u1ea1o th\u01b0 m\u1ee5c L\u1ecbch nh\u1eafc nh\u1edf sinh nh\u1eadt
+BirthdayReminder_noContactsWereChecked = Kh\u00f4ng ch\u1ecdn s\u1ed1 li\u00ean h\u1ec7 n\u00e0o
+BirthdayReminder_couldNotDeleteCalendar = Kh\u00f4ng th\u1ec3 x\u00f3a th\u01b0 m\u1ee5c L\u1ecbch nh\u1eafc nh\u1edf sinh nh\u1eadt
+BirthdayReminder_formatIncorrect = Ng\u00e0y sinh nh\u1eadt ph\u1ea3i c\u00f3 \u0111\u1ecbnh d\u1ea1ng "YYYY-MM-DD"
+
+#
+# Preferences dialog
+#
+BirthdayReminder_zimletPreferences = Tr\u00ecnh nh\u1eafc sinh nh\u1eadt: T\u00f9y ch\u1ecdn
+BirthdayReminder_PleaseNote = L\u01b0u \u00fd: Zimlet n\u00e0y s\u1ebd t\u1ef1 \u0111\u1ed9ng kh\u1edfi t\u1ea1o m\u1ed9t l\u1ecbch g\u1ecdi l\u00e0 <b>'{0}'</b> <br/> \u0111\u1ec3 l\u01b0u m\u1ecdi l\u1eddi nh\u1eafc nh\u1edf sinh nh\u1eadt.
```

Comment:
English source doesn't have them either and it's the same for all other languages.
Should I fix them in all files + English? Is there a ticket for this?